frontends/infinite-corridor/src/api/learning-resources/hooks.test.tsx

```diff
@@ -75,68 +77,29 @@ test("useResource rejects with invalid resource type", async () => {
 })
 
 test("useCreateUserList invalidates userlist Listings", async () => {
```

Comment:
When we perform mutations (POST, PATCH, etc) we invalidate previously fetched queries, causing them to be refetched.

Here, we were calling `useCreateUserList` (a mutation) and some queries (e.g., `useUserListsListing`) and checking that the UserList listing was re-fetched after the mutation was made.

This approach proved to be a bit fragile and was also more effort to write, so I had previously re-written a bunch of other, similar tests to just check that `invalidateQueries` is called correctly by the mutation.

Here, this a similar update is made to this test.